File: C#/LocalSearchOptimization/Parameters/MultistartParameters.cs
using System;

namespace LocalSearchOptimization.Parameters
{
    public class MultistartParameters : OptimizationParameters
    {
        public int InstancesNumber { get; set; } = 1;

        public bool RandomizeStart = true;

        public int OutputFrequency { get; set; } = 100;

        public Type OptimizationAlgorithm;

        public OptimizationParameters Parameters;

        public MultistartParameters()
        {

        }

        protected MultistartParameters(MultistartParameters copy) : base(copy)
        {
            InstancesNumber = copy.InstancesNumber;
            RandomizeStart = copy.RandomizeStart;
            OutputFrequency = copy.OutputFrequency;
            OptimizationAlgorithm = copy.OptimizationAlgorithm;
            Parameters = copy.Parameters;
        }

        public override OptimizationParameters Clone()
        {
            return new MultistartParameters(this);
        }
    }
}

File: java/src/localsearchoptimization/parameters/MultistartParameters.java
package localsearchoptimization.parameters;

public class MultistartParameters extends OptimizationParameters {

    public int instancesNumber = 1;

    public int outputFrequency = 100;

    public Class optimizationAlgorithm;

    public OptimizationParameters parameters;

    public MultistartParameters() {

    }

    protected MultistartParameters(MultistartParameters copy) {
        super(copy);
        instancesNumber = copy.instancesNumber;
        outputFrequency = copy.outputFrequency;
        optimizationAlgorithm = copy.optimizationAlgorithm;
        parameters = copy.parameters;
    }

    @Override
    public MultistartParameters Clone() {
        return new MultistartParameters(this);
    }
}